content/docs/alerting/configuration.md

```diff
@@ -78,7 +78,9 @@ global:
   [ smtp_auth_identity: <string> ]
   # SMTP Auth using CRAM-MD5. 
   [ smtp_auth_secret: <secret> ]
-  # The default SMTP TLS requirement.
+  # The default SMTP TLS requirement. 
+  # TLS should be enabled for security reasons if using remote endpoint as smarthost.
+  # Disabling tls when using a remote endpoint as smarthost would throw unencrypted connection error.
```

Comment:
"Note that Go does not support unencrypted connections to remote SMTP endpoints".

Reply:
Done